samples/bluetooth/peripheral_hids_mouse/src/main.c

```diff
@@ -206,9 +206,13 @@ static void advertising_start(void)
 	bt_foreach_bond(BT_ID_DEFAULT, bond_find, NULL);
 #endif
 
-	advertising_continue();
+	k_work_submit(&adv_work);
```

Comment:
@KAGA164 I would suggest to just remove this handling. As you can see `advertising_start` calls `advertising_stop` because the advertising is already running. we could just leave it running. 
The host has handling of making sure the advertiser is running while there are connectable slots open.
Alternatively, BT_LE_ADV_OPT_ONE_TIME should be set, and then remove bt_le_adv_stop from this function.

Reply:
Do you mean to remove `k_work_submit` here or to remove `advertising_start` and turning on adv without stopping it because this is done in `adv_start `?

Reply:
I removed adv stop function because we use BT_LE_ADV_OPT_ONE_TIME  there